src/transformers/models/vits/tokenization_vits.py

```diff
@@ -172,11 +174,12 @@ def prepare_for_tokenization(
         filtered_text = self._preprocess_char(text)
 
         if has_non_roman_characters(filtered_text) and self.is_uroman:
-            logger.warning(
```

Comment:
This is my only qualm with the PR: it's currently not backwards compatible. 

Previously, if a user was omitting the uroman step and inputting non-roman characters, they would have only get a simple warning. Now, the code will error out and stop them from doing this (i.e. breaking backwards compatibility).

I would be in favour of keeping the `logger.warning` (rather than an `ImportError`) and advising the user to install the `uroman` package
```py
logger.warning(
    "Text to the tokenizer contains non-Roman characters. To apply the `uroman` pre-processing "
    "step automatically, ensure the `uroman` Romanizer is installed with: `pip install uroman`
    "Otherwise, apply the Romanizer manually as per the instructions: https://github.com/isi-nlp/uroman
)
```

Reply:
Sorry, didn't noticed the default behaviour, I have modified accordingly.
Now it will only give warning in case the `uroman` package is not installed.
Thanks for the inputs.

Reply:
Looks great after the changes!